FabricTestExample/ios/Podfile.lock

```diff
@@ -1564,8 +1564,8 @@ SPEC CHECKSUMS:
   RNReanimated: 2f33b2f8c79c456ce3858a7cd2e07460eb5f61e0
   RNScreens: 63fe8222c172a79f5c30dd1aefaeb369c6eb57b6
   SocketRocket: abac6f5de4d4d62d24e11868d7a2f427e0ef940d
-  Yoga: b9a182ab00cf25926e7f79657d08c5d23c2d03b0
+  Yoga: 348f8b538c3ed4423eb58a8e5730feec50bce372
 
 PODFILE CHECKSUM: 67b3d295da87c29349179e51bb3526b67059b646
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.14.3
```

Comment:
I'm wondering if we shouldn't introduce some policy of using chosen version of cocoapods to omit this sort of changes from Podfiles 🤔 what do you think?

Reply:
Yeah, would be nice to just hardcode it so it does not land in many PRs by mistake